codec/src/main/java/io/netty/handler/codec/CharSequenceValueConverter.java

```diff
@@ -126,12 +126,12 @@ public CharSequence convertTimeMillis(long value) {
 
     @Override
     public long convertToTimeMillis(CharSequence value) {
-        try {
-            return HeaderDateFormat.get().parse(value.toString());
-        } catch (ParseException e) {
-            PlatformDependent.throwException(e);
+        Date date = DateFormatter.parseHttpDate(value);
+        if (date == null) {
+            PlatformDependent.throwException(new ParseException(value.toString(), 0));
```

Comment:
Previously the ParseExcpetion included some other descriptive text (`throw new ParseException("header can't be parsed into a Date: " + value, 0);`). Can we be consistent and either include similar text here?